exporter/containerimage/exptypes/types.go

```diff
@@ -13,6 +13,7 @@ const (
 	ExporterInlineCache          = "containerimage.inlinecache"
 	ExporterBuildInfo            = "containerimage.buildinfo"
 	ExporterPlatformsKey         = "refs.platforms"
+	ExporterTypeKey              = "exporter.type"
```

Comment:
I think this logic should be removed now when we still only return one map as result.